ruby/Dockerfile

```diff
@@ -27,64 +27,15 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-FROM debian:latest
+FROM ruby:2.1
 
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN wget https://github.com/grpc/grpc/releases/download/release-0_5_0/libgrpc_0.5.0_amd64.deb
+RUN wget https://github.com/grpc/grpc/releases/download/release-0_5_0/libgrpc-dev_0.5.0_amd64.deb
+RUN dpkg -i libgrpc_0.5.0_amd64.deb libgrpc-dev_0.5.0_amd64.deb
 
-RUN apt-get update && apt-get install -y \
-  autoconf \
-  automake \
-  autotools-dev \
-  bison \
-  build-essential \
-  curl \
-  g++ \
-  gawk \
-  gcc \
-  git \
-  libc6-dbg \
-  libc6-dev \
-  libffi-dev \
-  libgdbm-dev \
-  libgtest-dev \
-  libgflags-dev \
-  libncurses5-dev \
-  libreadline6-dev \
-  libsqlite3-dev \
-  libssl-dev \
-  libtool \
-  libyaml-dev \
-  make \
-  patch \
-  pkg-config \
-  procps \
-  sqlite3 \
-  unzip \
-  zlib1g-dev && apt-get clean
+RUN mkdir proto-bins
+RUN wget https://github.com/grpc/grpc/releases/download/release-0_5_0/proto-bins_0.5.0_linux-x86_64.tar.gz
+RUN tar xf proto-bins_0.5.0_linux-x86_64.tar.gz -C proto-bins
```

Comment:
As discussed, the tar.gz archive has been updated; so this line will need to be updated